### PR TITLE
Fix mark as spam display

### PR DESCRIPTION
**Pull request by GretaD**

### CODE REVIEW

File: lib/Db/Message.php [hunk 1]
```diff
@@ -84,8 +84,8 @@ class Message extends Entity implements JsonSerializable {
 		'flagged',
 		'seen',
 		'forwarded',
-		'junk',
-		'notjunk',
+		'$junk',
```
Comment (GretaD) [inline lib/Db/Message.php:87]: just testing stuff

**Comment by st3iny** [comment]
I pushed necessary backend changes and fixed the thread envelope menu.

**Review by miaulalala — APPROVED** [review]

**Comment by GretaD** [comment]
i will check the backport problem

**Review by ChristophWurst — COMMENTED** [review]
For completeness the logic discussed in our call that isn't in the diff was already there: https://github.com/nextcloud/mail/blob/f44abf87fc5a4b7407ccb205b2bbd8470b4bca3d/lib/Model/IMAPMessage.php#L739. That means old `junk` messages will map to `$junk`.

BUT old `notjunk` won't map to `$notjunk`.

**Pull request merged**